### DISCUSSION TITLE
Struggling to addd to Calibre - I must be doing something wrong - right?

### DISCUSSION THREAD

**Discussion by thephatmaster** [Leseratte10/acsm-calibre-plugin#59] [General]
I'm struggling with getting going with this plugin - any ideas what I'm doing wrong?

- I can't find it in Calibre's plugin repo; and
- If I try a manual install Calibre crashes when opening the zip

I'm using Calibre 6 on KDE Neon (Ubunto 22.04 base)

**Comment by Leseratte10** [comment]
In Calibre's repo it should be listed under "DeACSM". 

Not sure why it crashes with a manual install, though. Can you try creating a debug log?

Start calibre from the command line using `calibre-debug -g`, add the plugin, and post the resulting logs.

**Reply by thephatmaster** [reply]
Thanks,

Can't see it in the repo - actually - looking at the Calibre "crash" - the plugin did get installed via the zip. It seems Calibre tried to restart but couldn't, so likely a Calibre 6 issue.
